test/python/api_config.py

```diff
@@ -40,12 +40,13 @@ def get_api_key(username):
     api_key = auth_api.rotate_api_key(os.environ[CONJUR_ACCOUNT], role=f'user:{username}')
     return api_key
 
-def get_api_client(username='admin'):
+def get_api_client(username='admin', api_key=None):
     """
     Gets an authenticated ApiClient with the given
     username/password to be used with the testing setup
     """
-    api_key = get_api_key(username)
+    if not api_key:
```

Comment:
I don't know if this is the correct way to do this. The rotate API key endpoint provides the ability to rotate a hosts key as well, I think it would be cleaner to be able to specify a role (i.e. host in this case) then to have to pass in the API key. Based on how you use it it should still work fine, and fit better.